bodhi/server/scripts/skopeo_lite.py

```diff
@@ -0,0 +1,717 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2018 Red Hat, Inc. and others.
```

Comment:
Since you and I both work at Red Hat, I think we can drop the "and others" here for now. I usually add that when there are commits in the file from people who don't work at Red Hat.